worldedit-core/src/main/java/com/sk89q/worldedit/LocalSession.java

```diff
@@ -1164,4 +1172,15 @@ public boolean isNavWandItemDefault() {
     public void setLastDistribution(List<Countable<BlockState>> dist) {
         lastDistribution = dist;
     }
+
+    /**
+     * Call when this session has become inactive.
+     *
+     * <p>This is for internal use only.</p>
+     */
+    public void didBecomeIdle() {
```

Comment:
I would rather call this:
```suggestion
    public void onIdle() {
```